modules/tariffinfo.php

```diff
@@ -47,6 +217,33 @@
 
 $SESSION->save('backto', $_SERVER['QUERY_STRING']);
 
+// if selected tariff is phone tariff then load prefixes assigned to this tariff
+if ($tariff['type'] == 4) {
+	$tariff_rows = $DB->GetAll('SELECT DISTINCT
+											 g.id as group_id,
+											 g.name as group_name,
+											 p.prefix as prefix_number
+										FROM
+											(voip_tariff t LEFT JOIN voip_prefix_group g ON t.groupid = g.id)
```

Comment:
Czy to nie o to chodziło?
```sql
SELECT DISTINCT a.groupid AS group_id, g.name AS group_name,
    p.prefix AS prefix_number
  FROM voip_prefix_group_assignments a
  LEFT JOIN voip_prefix_group g ON g.id  = a.groupid
  LEFT JOIN voip_prefix p ON p.id = a.prefixid
```
Po co jest w tabeli **voip_tariffs** pole **groupid**?
Czyli voip_tariffs to tabela wiążąca rekordy taryf z tabeli **tariffs** z rekordami w tabelach **voip_prefix_groups** i **voip_prefixes**?